cfme/tests/services/test_service_customer_bz.py

```diff
@@ -115,3 +116,48 @@ def test_refresh_dynamic_field(appliance, import_datastore, import_data,
     view.fields(ele_label).fill("Password")
     # If Refresh works submit button will enable
     wait_for(lambda: not view.submit_button.disabled, timeout=7)
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1570152)])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize(
+    "import_data",
+    [DatastoreImport("bz_1570152.zip", "bz_1570152", None)],
+    ids=["sample_domain"],
```

Comment:
It would be better if we wait for this PR - https://github.com/ManageIQ/integration_tests/pull/9222/ to get merged.

I have added some changes in ```DatastoreImport```. We need to provide *description* of the domains we are importing.